packages/fxa-settings/src/components/Settings/PageRecentActivity/en.ftl

```diff
@@ -1,13 +1,32 @@
-## Recent Activity
+## Recent account activity
+## All strings except title indicate an event that occurred from the user's account
+## These are displayed as a list with the date when the event occured
 
-recent-activity-title = Recent Account Activity
+recent-activity-title-v2 = Recent account activity
 
-recent-activity-account-create = Account was created
-recent-activity-account-disable = Account was disabled
-recent-activity-account-enable = Account was enabled
-recent-activity-account-login = Account initiated login
-recent-activity-account-reset = Account initiated password reset
-recent-activity-emails-clearBounces = Account cleared email bounces
+recent-activity-account-create-v2 = Account created
+recent-activity-account-disable-v2 = Account disabled
+recent-activity-account-enable-v2 = Account enabled
+recent-activity-account-login-v2 = Account login initiated
+recent-activity-account-reset-v2 = Password reset initiated
+recent-activity-emails-clearBounces-v2 = Email bounces cleared
+recent-activity-account-login-failure = Account login attempt failed
+recent-activity-account-two-factor-added = Two-step authentication enabled
+recent-activity-account-two-factor-requested = Two-step authentication requested
+recent-activity-account-two-factor-failure = Two-step authentication challenge failed
+recent-activity-account-two-factor-success = Two-step authentication challenge successful
+recent-activity-account-two-factor-removed = Two-step authentication removed
+recent-activity-account-password-reset-requested = Account requested password reset
+recent-activity-account-password-reset-success = Account password reset successful
+recent-activity-account-recovery-key-added = Account recovery key enabled
+recent-activity-account-recovery-key-challenge-failure = Account recovery key challenge failed
+recent-activity-account-recovery-key-challenge-success = Account recovery key challenge successful
+recent-activity-account-recovery-key-removed = Account recovery key removed
+recent-activity-account-password-added = New password added
+recent-activity-account-recovery-key-added = Password changed
+recent-activity-account-secondary-email-added = Secondary email address added
+recent-activity-account-secondary-email-removed = Secondary email address removed
+recent-activity-account-emails-swapped = Primary and secondary emails swapped
 
-## $date (Date) - Date recent activity was created
-recent-activity-created-at = { $date }
```

Comment:
@bcolsson I've removed this FTL string since we were only passing in a variable with no other text. 

Instead, I've used [Intl.DateTimeFormat]( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) to localize the date directly. I'm not sure if this is appropriate, and would appreciate your input. 

```
  const localizedDate = Intl.DateTimeFormat(navigator.language, {
    dateStyle: 'medium',
    timeStyle: 'short',
  }).format(new Date(createdAt));
  ```

Looks good in French, which is the only non-English language I can confidently check.

![image](https://github.com/mozilla/fxa/assets/22231637/fe32f385-999f-48bc-8fbb-9e93be3d01b6)


Here are a few other screenshots for dates in other languages. Please let me know if other locales would be helpful!

Hebrew (RTL)

![image](https://github.com/mozilla/fxa/assets/22231637/8ef994ba-4f15-45db-b0f0-f158f2a5493d)

Japanese (just shows as numeric?)

![image](https://github.com/mozilla/fxa/assets/22231637/96e69296-2123-4ada-b3d2-650c82a7c9d6)

Belarusian

![image](https://github.com/mozilla/fxa/assets/22231637/12ca4705-92ab-4cc2-be79-038326518ddd)

Reply:
This is essentially what Fluent [already does](https://projectfluent.org/fluent.js/bundle/classes/fluentdatetime.html). 

I don't have any issues with removing this as a string, given how it's used and the context around it, this change makes sense.

Outputs look good. (Japanese has many ways to express a date, but in this context this output is what I would expect.)